docs/10-relevant_links.md

```diff
@@ -0,0 +1,26 @@
+---
+title: zkVerify Hub
+---
+
+# zkVerify Hub
+
+
+## zkVerify Links
+
+| Product | Link |
+| --- | --- |
+| zkVerify Block Explorer | https://testnet-explorer.zkverify.io/v0 |
+| Monitoring | https://testnet-telemetry.zkverify.io |
+| Documentation | https://docs.zkverify.io/ |
+| RPC Gateway | https://testnet-rpc.zkverify.io, wss://testnet-rpc.zkverify.io |
+| Proof Submission via PolkadotJS | https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fnh-rpc.horizenlabs.io |
+| zkVerify Attestation Contract on Sepolia | `0x627efD0F38f67Edb9Dc69fa3B7e19208Fd41Eac7`
```

Comment:
Could we add the link for this as well ?

Reply:
https://sepolia.etherscan.io/address/0x627efD0F38f67Edb9Dc69fa3B7e19208Fd41Eac7

Reply:
Added - good callout!